### PR TITLE
[stable/external-dns] Add support for AWS Zone tags filter

### PR DESCRIPTION
**Pull request by alexgavrisco**
Signed-off-by: Alex Gavrisco <alexxg.tmd@gmail.com>

#### What this PR does / why we need it:

External DNS allows filtering zones by tags for AWS provider (via `aws-zone-tags` option), this PR allows passing `aws.zoneTags` value.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/external-dns/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.3.3
+version: 2.3.4
```
Comment (tompizmor) [inline stable/external-dns/Chart.yaml]: Following [semver versioning](https://semver.org/), as you are introducing a new feature to the chart, you should update the minor version and not the patch one. Please update to `2.4.0`

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/external-dns/Chart.yaml (hunk 1 above)
Comment (alexgavrisco) [reply stable/external-dns/Chart.yaml]: Done.

**Review by tompizmor — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**